README.md

```diff
@@ -81,6 +81,22 @@ Check out the document in the [website](https://wrtnlabs.io/agentica):
 - [⇲ API Documents](https://wrtnlabs.io/agentica/api)
 
 
+## How to Contribute?
+
+### 💻 Development
+
+- Clone this repository
+- Install dependencies using `pnpm install`
+
+### 📦 Build
+
+- Run `pnpm build` to build the project.
+
+### 🚀 Publish
+
+- Run `pnpm release` in local environment. This command bump the version and create git tag! Then, push the tag to the remote repository.
```

Comment:
I think contribution documentation exists for external contributors.
The command exists for insiders.

Reply:
wait it is open source actually.
Also we don't have any place to leave this information other than this repository.
Do you have any idea to add this information?

Reply:
"Deployment works like this"
How about a title like this?

Reply:
good!
Or could you write this in the appropriate place on my behalf

Reply:
I'll write it up when I have time.